CHANGELOG.md

```diff
@@ -6,7 +6,13 @@ here: [Community Features](https://github.com/SynthstromAudible/DelugeFirmware/b
 
 ## c1.2.0 Unnamed
 
```

Comment:
careful not to merge all these 1.2 items into release 1.1 

if this is merged into 1.1, only one of those items should be included and should be moved under c.1.1.0

"- Added a new default setting that controls which playback mode new slices of a kit will get. "